internal/commands/config/list_profiles_internal.go

```diff
@@ -1,21 +1,44 @@
 package config_internal
 
 import (
+	"slices"
+
+	"github.com/fatih/color"
+	"github.com/pingidentity/pingcli/internal/logger"
 	"github.com/pingidentity/pingcli/internal/output"
 	"github.com/pingidentity/pingcli/internal/profiles"
 )
 
 func RunInternalConfigListProfiles() {
+	l := logger.Get()
+
 	profileNames := profiles.GetMainConfig().ProfileNames()
 	activeProfile := profiles.GetMainConfig().ActiveProfile().Name()
 
 	listStr := "Profiles:\n"
+
+	slices.Sort(profileNames)
```

Comment:
This seems useful for the `ProfileNames()` function to do generally.

Reply:
Makes sense!